src/openshiftCommonUI.module.js

```diff
@@ -5,6 +5,15 @@
  *   Base module for openshiftCommonUI.
  */
 angular.module('openshiftCommonUI', [])
+// Sometimes we need to know the css breakpoints, make sure to update this
+// if they ever change!
+.constant("BREAKPOINTS", {
+  screenXsMin:  480,   // screen-xs
+  screenSmMin:  768,   // screen-sm
+  screenMdMin:  992,   // screen-md
+  screenLgMin:  1200,  // screen-lg
+  screenXlgMin: 1600   // screen-xlg
+})
```

Comment:
Hmm... Some if this is defined in patternfly:

  patternfly.pfBreakpoints = {
    'tablet': 768,
    'desktop': 1200
  };

Maybe the other values ought to be defined there as well. 
Until then, the patternfly constant is used in the catalog repo already(https://github.com/openshift/origin-web-catalog/blob/master/src/components/saas-list/saas-list.controller.ts#L51-L52). We may want to update that to use this constant instead.